sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractKeyPhrasesActionResult.cs

```diff
@@ -2,25 +2,38 @@
 // Licensed under the MIT License.
 
 using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
 using Azure.AI.TextAnalytics.Models;
 
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
-    /// ExtractKeyPhrasesActionResult
+    /// The result of the execution of a <see cref="ExtractKeyPhrasesAction"/> on the input documents.
     /// </summary>
     public class ExtractKeyPhrasesActionResult : TextAnalyticsActionResult
     {
-        internal ExtractKeyPhrasesActionResult(ExtractKeyPhrasesResultCollection result, DateTimeOffset completedOn, TextAnalyticsErrorInternal error) : base(completedOn, error)
+        private readonly ExtractKeyPhrasesResultCollection _documentsResults;
+
+        internal ExtractKeyPhrasesActionResult(ExtractKeyPhrasesResultCollection result, DateTimeOffset completedOn, TextAnalyticsErrorInternal error)
+            : base(completedOn, error)
         {
-            Result = result;
+            _documentsResults = result;
         }
 
         /// <summary>
-        /// Results
+        /// Gets the result of the execution of a <see cref="ExtractKeyPhrasesAction"/> per each input document.
```

Comment:
nit: of "an"

Reply:
Thank you for fixing all my typos or bad English :(